packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/sagas.ts

```diff
@@ -1292,17 +1292,18 @@ export default ({
   const showModal = function * ({ payload }: ReturnType<typeof A.showModal>) {
     const { cryptoCurrency, orderType, origin } = payload
     const latestPendingOrder = S.getSBLatestPendingOrder(yield select())
-    const bankAccount = yield call(
-      getBankInformation,
-      latestPendingOrder as SBOrderType
-    )
+
     yield put(
       actions.modals.showModal('SIMPLE_BUY_MODAL', { origin, cryptoCurrency })
     )
     const fiatCurrencyR = selectors.core.settings.getCurrency(yield select())
     const fiatCurrency = fiatCurrencyR.getOrElse('USD')
 
     if (latestPendingOrder) {
+      const bankAccount = yield call(
+        getBankInformation,
+        latestPendingOrder as SBOrderType
+      )
```

Comment:
we should not call this one before since there might be a fresh wallet

Reply:
LGTM